docs/markdown/Using Pants/using-pants-in-ci.md

```diff
@@ -15,9 +15,16 @@ Directories to cache
 
 In your CI's config file, we recommend caching these directories:
 
-- `$HOME/.cache/pants/setup`: the initial bootstrapping of Pants.
-- `$HOME/.cache/pants/named_caches`: caches of tools like pip and PEX.
-- `$HOME/.cache/pants/lmdb_store`: cached content for prior Pants runs, e.g. prior test results.
+- `$HOME/.cache/pants/setup`: The initial bootstrapping of Pants.
+- `$HOME/.cache/pants/named_caches`: Caches used by underlying tools like Pip and PEX.
+
+If you're not using a fine-grained remote caching service, such as the one provided by
```

Comment:
This should link to the page on remote caching. And we may want to update that page too to mention Toolchain (fine in followup)

Reply:
Done